RecipeMatch/Views/DraggableView.m

```diff
@@ -33,60 +33,47 @@ @implementation DraggableView {
 @synthesize title;
 @synthesize recipeId;
 @synthesize recipeImage;
+@synthesize imageUrl;
 @synthesize url;
 @synthesize ingredients;
 @synthesize time;
 @synthesize servings;
 @synthesize overlayView;
+@synthesize detailsBtn;
 
 - (id)initWithFrame:(CGRect)frame
 {
     self = [super initWithFrame:frame];
     if (self) {
         [self setupView];
-        
 #warning placeholder stuff, replace with card-specific information {
-        title = [[UILabel alloc]initWithFrame:CGRectMake(0, 20, self.frame.size.width, 20)];
-        title.text = @"no info given";
-        [title setTextAlignment:NSTextAlignmentCenter];
-        title.textColor = [UIColor blackColor];
         
         self.backgroundColor = [UIColor whiteColor];
 #warning placeholder stuff, replace with card-specific information }
         
-        recipeImage = [[UIImageView alloc] initWithFrame:CGRectMake(0,0,0,0)];
+        title = [[UILabel alloc]initWithFrame:CGRectMake(0, self.frame.size.height - 80, self.frame.size.width, 60)];
+        title.lineBreakMode = NSLineBreakByWordWrapping;
+        title.numberOfLines = 0;
+        [title setTextAlignment:NSTextAlignmentCenter];
+        title.textColor = [UIColor blackColor];
+        [[self title] setFont:[UIFont systemFontOfSize:16]];
+        
+        int imgSize = 300;
+        recipeImage = [[UIImageView alloc] initWithFrame:CGRectMake((self.frame.size.width-imgSize)/2,10,imgSize,imgSize)];
         recipeImage.translatesAutoresizingMaskIntoConstraints = NO;
         [recipeImage setContentMode:UIViewContentModeScaleAspectFit];
+        recipeImage.layer.masksToBounds = YES;
+        recipeImage.layer.cornerRadius = 15;
```

Comment:
This will work, 
just so you know that you can theoretically (if you prefer), create a new UIView class with XIB and then using autolayout set it as you want, and then add it as a subview here.
again, totally not needed, it was just for your information